themes/wilson_theme_starterkit/src/scss/components/jump-nav.scss

```diff
@@ -0,0 +1,90 @@
+.jump-nav {
+  @apply sticky;
+  @apply top-[-1px];
+  @apply z-40;
+  @apply transition-colors;
+  @apply transition-shadow;
+  @apply duration-300;
+
+  &.is-sticky {
+    @apply bg-white;
+    @apply overflow-hidden;
+    @apply z-[60];
+    @apply shadow-xl;
+
+    nav {
+      @apply drop-shadow-none;
+    }
+  }
+
+  nav {
+    @apply drop-shadow-2xl;
+    @apply overflow-x-scroll;
+
+    &::-webkit-scrollbar {
+      display: none;
```

Comment:
Is there anything required to handle non webkit browsers?

Reply:
Added specific styles for edge and firefox.